src/pudl/transform/eia860.py

```diff
@@ -897,12 +873,6 @@ def _core_eia860__boilers(
         b_df.loc[b_df.report_date.dt.year < 2012, "efficiency_50pct_load"] / 100
     )
 
-    b_df = (
-        pudl.metadata.classes.Package.from_resource_ids()
-        .get_resource("core_eia860__scd_boilers")
-        .encode(b_df)
-    )
-
```

Comment:
Removed the encoding step because it's the last part of the transform, and now happens in the harvesting process for this table.

Reply:
same question as above (re the `eia.core_eia860__assn_boiler_generator`

Reply:
How are you identifying this as something that might need to be independently encoded?